ocaml/xapi/xapi_globs.ml

```diff
@@ -945,6 +945,8 @@ let repository_gpgkey_name = ref ""
 
 let repository_gpgcheck = ref true
 
+let bundle_file_max_size = ref (Int64.of_int (500 * 1024 * 1024))
```

Comment:
This appears to be used only in test code, is that intended? In that case it shouldn't be in xapi_globs.

Reply:
It will also be used in another future PR that I have raised.
Let me remove this configure and move it into that PR.

Reply:
Or how about consider it a constraint for all tar unpacking operations, then we should use this `bundle_file_max_size` directly instead of passing it as a parameter to `bundle_file_max_size`.